doc/release_notes.rst

```diff
@@ -10,11 +10,13 @@ Upcoming release
 ================
 
 Please add descriptive release notes like in `PyPSA-Eur <https://github.com/PyPSA/pypsa-eur/blob/master/doc/release_notes.rst>`__.
-E.g. if a new rule becomes available describe how to use it `snakemake -j1 run_tests` and in one sentence what it does.
+E.g. if a new rule becomes available describe how to use it `make test` and in one sentence what it does.
 
 **New Features and Major Changes**
 
+* The workflow configuration now supports incremental changes to the default configuration in the `config.yaml` and configfiles passed to snakemake via `--configfile myconfig.yaml`. Therefore the user may now only include settings in their `config.yaml` which differ from the default configuration. `PR #1053 <https://github.com/pypsa-meets-earth/pypsa-earth/pull/1053>`_
```

Comment:
Maybe add 1-2 more sentences, what the benefit/purpose of this is?

Reply:
yes good one